### ISSUE TITLE
class TerminalBlockExtentions is misspelled, it should be Extensions (ie. s/nt/ns/)

### ISSUE THREAD

**Issue by thatpixguy** [KeenSoftwareHouse/SpaceEngineers#595]
The class is defined [here](https://github.com/KeenSoftwareHouse/SpaceEngineers/blob/master/Sources/Sandbox.Common/ModAPI/Ingame/IMyTerminalBlock.cs#L37)

The spelling is defined [here](https://www.merriam-webster.com/dictionary/extension) :P

**Comment by mccorkle** [comment]
A good find, but I'm not sure if this code is relevant any more as keen isn't updating nor actively monitoring this repo any more.

**Comment by thatpixguy** [comment]
Yeah I noticed that a little while after reporting. I also learned that there is a way to get at what I was using without referencing that class explicitly (which is how I noticed the misspelling).
